### ISSUE TITLE
Video NFT Design Brief

### ISSUE THREAD

**Issue by bedeho** [Joystream/atlas#1329]
# Purpose

Allow creators to monetize their most impactful content by granting their most passionate community members to be recognized as the owners of individual videos. This ownership is itself a non-fungible token, in that users can transfer or auction away this ownership to new owners, and ownership of one video is distinct from owning another, hence the non-fungible modifier.

# Background

> This section is extremely important to understanding the core model of how NFTs unlock value, and it should be critical in guiding design choices and research.

A passionate follower of some creator or topic may find significant value in being able to associated themselves with related creative content. They key is that the association is very public and visible, and thereby allows the owner to signal to their community and related audiences something about who they are, the resources they have - financial, social and human, in the latter for example by having discerning taste or judgement. This behaviour is already very prevalent in lots of social arenas in the off-line world, in particular among collectors. People collect stamps, coins, art, cars, sneakers, etc., and motivations span:

1. inherent enjoyment in collecting anything durable, sort of like a hobby, where you also get to curate a collection
2. a tasteful way to demonstrate you have access to resources like time, money and information
3. showing you have good judgement and taste
4. owning a piece of history in some way
5. making a good investment

Video NFTs play into all of these objectives, and arguably substantially amplify your ability to achieve those goals through the following

- **More public:** you and your assets are visible for everyone to see, and is a highly entertaining and viral medium, e.g. unlike a painting hanging on a wall in your house.
- **More financial:** anyone can bid on and buy your assets 24/7/365, giving you increased liquidity and greater price discovery. This makes it particularly valuable to be early, with good judgement, before something goes viral or gains attention, and perhaps even makes it valuable to be able to market something and make it valuable.

Hence the name of the game here is really to empower the owner to be visible as the owner, to support all these goals, and to make it easy to transact in this ownership. For the creator the value is in monetising the exclusive ownership status over their content.

**Critically, owning a video, in the sense described herein, is not a claim on the intellectual property, revenue or any other rivalrous property of using the actual content itself.**

# Resources

Here are some additional resources to learn about this concept

- **Overview of NFTs:** https://a16z.com/2021/04/02/nfts-readings-resources/
- **How NFTs can allow long tail creators to monetise:** https://cdixon.org/2021/02/27/NFTs-and-a-thousand-true-fans
- **Five New Frontiers for NFTs:** https://medium.com/electric-capital/5-new-frontiers-for-nfts-c77de367631d
- **Example NFT plans on BitClout:**
  - https://docs.google.com/presentation/d/1iklCqm85gjnqv_CyTG7-vXMBmSG1yhtBE_-oexUGZ0A/edit#slide=id.ge30f8df505_0_0
  - https://docs.bitclout.com/bitclout-nfts

# System

`This can still be changed if good opportunities are identified which are not too hard to implement`.

https://joystream.gitbook.io/joystream-handbook/subsystems/video-nfts

Perhaps the most important design decision in the system, which has very material implications for how it can be used, is the limitation that you can only issue 1-of-1 edition NFTs for a video. This decision was made largely because it was simpler on the runtime side, and that the 1-of-1 mode seems to be the only ownership mode where its very easy to connect ownership of a video to the primary product experience itself. This may be wrong, some exploration and thinking is needed here, but it may still be the case that we should stick to 1-of-1 given the possible complexity of executing on this now,

# Constraint

This functionality is supposed to be live on testnet, before mainnet, which has at least two key implications for the product design

1. We cannot 100% guarantee that any NFTs issued and transacted will be transferred over from testnet to the main network, and we cannot guarantee that the history of any ownership that is transferred over will have full integrity.
2. There is no easy way reliable way for people to obtain the sufficient amount of $tJOY (testnet $JOY) for the purpose of participating in auctions or buying NFTs in general. We in particular want to strongly discourage people beginning to conduct any secondary market transactions for $tJOY in order to purchase NFTs or do anything else on the testnet. This appears to not have been a problem so far, but we want to maintain that status quo. It is not clear how to deal with these two conflicting objectives.

It is important that we design the product in a way where we communicate clearly to the end users at critical junctions of the key user flows, in order to not avoid any misunderstandings. At the same time, we have to be careful to not over-invest in how to build the product for testnet.

# User Stories

## Channel Owner

as a channel owner I would like to

- issue an NFT for a video while publishing it.
- issue an NFT for a video that does not have an NFT issued.
- see all NFTs I have issued and their state.
- get notified when I earn a royalty from a transaction of an NFT I issued.

## NFT Owner

An NFT owner is anyone owning a given NFT, possibly including the channel owner.

As an NFT Owner I would like to

### Open Auction

- start an open auction.
- cancel an open auction.
- get notified when someone bids in an open auction.
- get notified when someone buys using buy now in an open auction.
- be made aware when a bid in an open auction is about to become withdraw-able.
- get notified when someone withdraws their bid in an open auction.
- accept an open auction bid.

### English Auction

- start an english auction.
- cancel an english auction.
- get notified when someone bids in an english auction.
- get notified when someone buys using buy now in an English auction.

### Offer

- offer the NFT to another member.
- cancel an outstanding offer I have made to another member.
- get notified when someone I offered an NFT to accepts.

### Buy Now

- enable buy now.
- cancel buy now.
- get notified when someone buys using buy now when in buy now mode.

### General

- see all NFTs I own.
- I would like to see the status and history of a particular NFT I issued.

## Content Viewer

As a content viewer I would like to

- see the status of the NFT for a given video.

## Collector

A collector is someone with a general interest in purchasing, collecting and trading NFTs, motivated by a mixture of the incentives outlined above (1-5).

as a collector I would like to

### Open Auction

- bid on an open auction.
- cancel bid in an open auction.
- get notified if my open auction bid is accepted.
- get notified if my open auction bid is superseded by a higher bid.

### English Auction

- bid on an english auction.
- get notified when the english auction ends while I have highest bid.
- get notified if english open auction bid is superseded by a higher bid.
- settle an english auction where I won.

### Buy Now

- buy an NFT in buy now.

### Offer

- get notified when someone initiates an offer in my favour.
- accept a pending offer in my favour.

### General

- get notified when there is any important state change in an NFT I am interested in.
- stop getting notified about state changes in an NFT I previously was interested in.
- see all ongoing auctions.
- see a list of the largest recent transaction.
- see a list of members ranked by collection size.
- see a list of members ranked by most valuable collections.
- see a list of top collectors on a given channel.
- see ongoing auctions for a given channel.

# Execution

Just to add some context, the feature scope is huge, like gigantic, we do not need to tackle it all at once, and probably should not, as it will take too long to go into production, so you should def. try to identify some MVP feature set that would be the goal for the first sprint.

**Comment by kdembler** [comment]
@bedeho Could you maybe specify what do you have in mind when you mention "get notified". Do you imagine this as an in-app notification center/email notification/something else? Or nothing specific on your mind?

Seems we will need to introduce concept of viewer-only membership. Currently we only let users create memberships as part of the Studio flow, it seems this will need to change. We probably also need to somehow differentiate those from "gateway accounts" we may want to introduce in the future?

**Comment by bedeho** [comment]
> Could you maybe specify what do you have in mind when you mention "get notified". Do you imagine this as an in-app notification center/email notification/something else? Or nothing specific on your mind?

Yes, exactly. I imagine we can support this, as we are also planning to do in Pioneer 2, by simply querying certain event types, and filtering on dates which correspond to some local storage timestamp of when we last showed notification counts. We may also want to have local storage bank of event ids which enumerates exactly those events the user has engaged with. Some variation of this should be workable.

> Seems we will need to introduce concept of viewer-only membership. Currently we only let users create memberships as part of the Studio flow, it seems this will need to change. We probably also need to somehow differentiate those from "gateway accounts" we may want to introduce in the future?

All correct, I have another issue I have written, but not published yet, which tries to tie all of the upcoming Atlas enhancements together in a coherent way for the benefit of the design and tech team, so we can plan and discuss accordingly. I will let you know when all the materials are public. Actually, even the handbook for the NFT stuff is not done yet, so this is still WIP.

**Comment by Lenorette** [comment]
@bedeho together with @toiletgranny we went through this NFT design brief, [Restricting scope issue](https://github.com/Joystream/atlas/issues/1374) and [Membership vs Gateway account issue](https://github.com/Joystream/atlas/issues/1388) and we prepared a spreadsheet with possible MVP content based on what you've proposed.

https://docs.google.com/spreadsheets/d/1AcS4mLXo8ybMwmBx93jwhpqtnEhn4nIVKokTOHtiAe4/edit#gid=0

As you'll see in the spreadsheet we mostly agree with you on the MVP scope. 
We think that most content could be connected to the Viewer space, and only issuing NFT would be included in Publisher space. 
Please check it out and let us know if we understood correctly account types / user stories / experiences / pages in which new content will be included.

We also left few questions for you in "Questions" tab, to clarify some things.  

As mentioned in Monday weekly we could have a call tomorrow (Wednesday) to talk it through if you want.

**Comment by bedeho** [comment]
# Scope

I think we cannot avoid having these stories as well.

1. We are missing the ability to actually bid in an open auction.
2. We are missing the ability to cancel bid in an open auction.
3. We are missing the ability to issue an NFT on an existing video.
4. We are missing the ability for a viewer to see the status of the NFT for a given video, at least that it exists.
5. We are missing get notified when someone bids in an open auction.
6. We are missing get  notified when someone buys using buy now in an open auction.

I realize these were not explicitly addressed in the scope restriction issue, but I think they more or less have to be included by implication of what we are keeping in scope.

# Questions

> How the process of issuing NFT should look like, and 

I will try to flesh out this system specification asap, as it will explain more functional details of the system

https://joystream.gitbook.io/joystream-handbook/subsystems/video-nfts

Please let me know what time frame you need on this, would next Monday be too late?

> why it shouldn’t be possible from the beginning while uploading new video?

Just to simplify, but we can review later if that simplification is not so useful.

> Would it be possible to connect one gateway account to one membership? We could have one flow for creating an account. Firstly basic info for creation of a gateway account, with an option to add polkadot extension and with it membership account. Additionally if user in the beginning creates only gateway account he could be able to “upgrade” to membership account by connecting wallet (polkadot) later on in settings. What do you think about it?

This was a bit complex to fully parse here, but in general I would try to avoid introducing any form of explicit gateway account at this time, and as part of the NFT design sprint. I don't think we need to couple them, and gateway accounts are not urgently needed at this time. Let us discuss on next team meeting to get on the same page.

> Could you tell us why there can be more than one membership created on the account (polkadot). Is it/will it really be useful for users?

A user could have more than one membership per (blockchain) account, or multiple memberships, each associated with a distinct blockchain account. Having more than one membership is useful, although not required, however it is not per say particularly important how memberships are associated with accounts, this is a distinct user choice.

Let us discuss during team meeting if this is not clear.

> "Is it possible to issue an NFT without being a channel owner? 
If not, I think this might be a duplicate of the third user story on the list: ""As a Channel owner, I want to see all NFTs I have issued and their state."""

You must be the channel owner.

**Comment by Lenorette** [comment]
### Scope
I think we had marked almost all of stories you've listed above, in the spreadsheet:

> 1. We are missing the ability to actually bid in an open auction.
> 2. We are missing the ability to cancel bid in an open auction.

 line 26 & 27

> 3. We are missing the ability to issue an NFT on an existing video.

If I understand correctly it's the same thing as line 2 (issue an NFT for a video that does not have an NFT issued)

>4. We are missing the ability for a viewer to see the status of the NFT for a given video, at least that it exists.

line 25

>5. We are missing get notified when someone bids in an open auction.

Notifications, Owner line 8,11; Viewer line 28, 29

>6. We are missing get notified when someone buys using buy now in an open auction.

line 9. Ok, this one wasn't marked as MVP yesterday. So with this one I'm a little confused right now, previously you've mentioned that we shouldn't include "Buy now", or by that you've ment option to only "Buy now" without any auction. And for an open auction we would have similar format to that Open sea has for eg. [here](https://opensea.io/assets/0x0b0b186841c55d8a09d53db48dc8cab9dbf4dbd6/573), so user can bid but also buy now?

### Division of work

We were also thinking how to divide NTF work between us when Kuba will be back, and here is our initial proposition:
- Kuba - changes in Publisher view
- Adam - changes in Video page
- Nina -  new page "My NFTs", where user would be able to see all NFTs created/owned/sold, manage them (start/cancel auctions)

### Questions 

> I will try to flesh out this system specification asap, as it will explain more functional details of the system
https://joystream.gitbook.io/joystream-handbook/subsystems/video-nfts
Please let me know what time frame you need on this, would next Monday be too late?

I think Monday will be fine :)

**Comment by iorveth** [comment]
> We are missing get notified when someone buys using buy now in an open auction.

> line 9. Ok, this one wasn't marked as MVP yesterday. So with this one I'm a little confused right now, previously you've >mentioned that we shouldn't include "Buy now", or by that you've ment option to only "Buy now" without any auction. And for an >open auction we would have similar format to that Open sea has for eg. here, so user can bid but also buy now?

@Lenorette , Auction related `buyNow` is different from regular `buyNow`. 
Regular `buyNow` is basically a sell order for nft outside of auction.  
Whereas auction `buyNow` is an optional feature, which allows you to instantly complete any auction at `buyNow` price.

https://joystream.gitbook.io/joystream-handbook/subsystems/video-nfts#start-nft-auction

**Comment by Lenorette** [comment]
@iorveth all clear now, thanks for clarification

**Comment by bedeho** [comment]
Here are my (delayed) replies to outstanding questions here: https://docs.google.com/spreadsheets/d/1AcS4mLXo8ybMwmBx93jwhpqtnEhn4nIVKokTOHtiAe4/edit#gid=1340391168

> Is Atlas a a video platform with NFT marketplace funcionalities added on top of it, or is it an NFT marketplace that also happens to be a video sharing platform?

I think we briefly coverd this on a call, but it is primarily a content consumption & publishing product, where NFTs are a dimension of the experience meant to enrich the social dynamics and monetization of publishing.

> NFT collections in Atlas? Is this something that will ever exist?

There is no explicit concept of NFT _collections_ in the system, but obviously, each member may own more than one NFT, hence they in a sense own a collection of NFTs. Aagain, here I am talking about Joystream as a system fundamentally, not Atlas specifically.

Please ask for clarification if this was not sufficiently focused answer.

> Just to confirm: a video, as an NFT, is issued by a Member, not Channel, right?

No, it is issued by the owner of a channel, on a video in that channel. Recall that a channel can be owner by a member, but it can also be owned by a curator group! of which there are a finite number at any time.

> Video title vs NFT title — is it the same?

Conceptually, owning the NFT for the video is meant to be recognised as thew owner of the video, so it's not clear what it would mean for an NFT to have a separate title. However, and this is very important, there is significant room for creative exploration here. In the concept design done earlier, each NFT had a user provided message, which could then be displayed with the video in certain cases. I thought that was very cool, but that is just a random example.

> What currency/currencies are we going to use for NFTs? ETH, tJOY?

NFTs are paid for with JOY, or testnet JOY (tJOY) while on testnet.

> A few command line based questions (eg. will I be able to list my NFT as a buy now through a command line, if so, what should the experience look like on eg. on viewer?)

I did not really understand this one. We will likely _eventually_ have full support for NFTs in our command line tool, however that is only meant for developers and power users.

> English Auction vs Open Auction? Have you seen any examples of marketplaces with both auction types?

I think I have seen at least one, but can't remember now.

> Following other Members on Viewer?

Is the question whether we should support this? That is hard to say, as it's not obvious to me what that would imply. Would it imply learning about when the they buy an NFT for example? That sounds interesting, but certainly outside of the narrow scope for MVP. If you want to suggest this, please a separate proposal which fleshes this out a bit.

> Just to confirm once a video becomes an nft it can not be deleted anymore.

Excellent question! Yes this is the current functionality, however it's a complex topic, so it may need to be changed or treated differently in the future.

**Comment by kdembler** [comment]
As a complete sidenote, should we create some kind of icon/symbol for JOY token to improve display of prices? I think ETH is a great example, it has both icon and symbol that can be used:
<img width="72" alt="Screenshot 2021-10-11 at 12 49 57" src="https://user-images.githubusercontent.com/12646744/136778365-2b445457-8564-4b00-9c9b-74878dde5493.png">
<img width="153" alt="Screenshot 2021-10-11 at 12 50 04" src="https://user-images.githubusercontent.com/12646744/136778385-b7a4c5a2-9fbc-4a0d-a9b3-3074e5bf052e.png">

Think this looks much cooler than for example `420 ETH`

**Comment by Lenorette** [comment]
@bedeho Thank you for the answers 

> _NFT collections in Atlas? Is this something that will ever exist?_
> 
> There is no explicit concept of NFT collections in the system, but obviously, each member may own more than one NFT, hence they in a sense own a collection of NFTs. Aagain, here I am talking about Joystream as a system fundamentally, not Atlas specifically.

Ok, I think we weren't to clear about what do we mean by Collections.
We had more in mind if Publisher will be able to create some sort of NFT Collection, something like [cryptopunks](https://opensea.io/assets/cryptopunks) or [robotos](https://www.robotos.art/). 
It would be like a group/playlist of videos that belong to one collection, and users (collectors) could try to get as many as they can from one collection. We don't have it in scope of MVP either way, but it is something to consider for future. 

> _Just to confirm: a video, as an NFT, is issued by a Member, not Channel, right?_
> 
> No, it is issued by the owner of a channel, on a video in that channel. Recall that a channel can be owner by a member, but it can also be owned by a curator group! of which there are a finite number at any time.

Could you tell me more about how the curator group actually works? It's the first time I'm hearing about it, and I didn't see any specific flows for that. 

1. Is it a channel somehow owned by multiple owners, or multiple people have an access to one membership?
2. How do they upload videos and manage this account? 
3. What is the reason for having curator group managing one channel?
3. How would it affect NFT (it's connected to 1st question, how it actually works), how would they divide JOYs earned, ect..

**Comment by bedeho** [comment]
> Ok, I think we weren't to clear about what do we mean by Collections.
We had more in mind if Publisher will be able to create some sort of NFT Collection, something like cryptopunks or robotos.
It would be like a group/playlist of videos that belong to one collection, and users (collectors) could try to get as many as they can from one collection. We don't have it in scope of MVP either way, but it is something to consider for future.

If I understand this correctly, what you are saying is that a collection in this sense is just the creator saying "these X videos belong together", it does not mean they are owned together as one group of videos, which can be bought/sold/transferred, as such?

If so, then it would seem to me that this use case only really works when there is more than 1 edition of each NFT, otherwise, the olny way to actually have a full collection is to get the exact single version of each video in the entire collection. It seems this kind of process can easily get stuck, with lots of bargaining costs and communication complexity.

If this is correct, then it seems we could at least not easily accommodate this, as we only support single edition NFTs. Its not impossible to change this, but it would require significant effort, so I would like to understand the full list of benefits before we decided to make such a change in the blockchain.

> Could you tell me more about how the curator group actually works? It's the first time I'm hearing about it, and I didn't see any specific flows for that.

Yes, the curators do their job through separate tooling, not through Atlas. You can think of them as employees of the DAO itself, who have the task of
- making sure content published by others follow the rules of the DAO, and they can even remove content.
- own and operate dedicated channels for the DAO, sort of like official channels. These channesl can publish videos & issue NFTs, just like normal channels.

> Is it a channel somehow owned by multiple owners, or multiple people have an access to one membership?

There are a set of different groups at any given time, called _curator groups_.  There is a dedicated actor, called the _content directory lead_. Who is responsible for creating and managing membership in these groups. Curator groups can have _workers_ as members, specifically workers in a system called the _content directory working group_. Think of this as a department in the organisation where people who are curators work (they get a salary, can get fired, etc.). Workers are just members, in the sense you are already familiar with, who have a job in this department. There are many such departments in the system for different purpose. In principle, the same member can actually be hired in more than one worker role in the same department, including in the content directory working group. The lead is responsible for adding and removing workers from each curator group. Now, as a curator group member, you can do things like create channels, operate channels, publish videos in group channels, .etc. If a worker is removed from a given curator group, then they can no longer manage the channels owned by this group.

I know this was rather complex, and we actually had very complete documentation on this, however it was for the previous version of our content directory, it will be a moment before we can update that. If you just want to understand the idea of these departments directly, you can check this part of the handbook, but it should not be required for you to be able to do the work at hand
https://joystream.gitbook.io/joystream-handbook/governance/working-groups

> How do they upload videos and manage this account?

At the core level of the system, same as normal members, however in terms of tooling they do not use Atlas!

> What is the reason for having curator group managing one channel?

They are the means by which the platform, that is the DAO, itself can have direct ownership over content and content publishing. Think of the analogy where YouTube itself owns content and chooses to publish it on the Youtube platform.

> How would it affect NFT (it's connected to 1st question, how it actually works), how would they divide JOYs earned, ect..

Hopefully this is clear.

One thing that is worth keeping in mind, it is that curators cannot participate in auctions and buyers/bidders. So the only way they can owne an NFT is if the NFT is owned by the channel, and they own the channel directly.

This part of the documentaiton will be fleshed out hopefully by early next week, right now its just stubbed
https://joystream.gitbook.io/joystream-handbook/subsystems/video-nfts

**Comment by Lenorette** [comment]
@bedeho 
> If I understand this correctly, what you are saying is that a collection in this sense is just the creator saying "these X videos belong together", it does not mean they are owned together as one group of videos, which can be bought/sold/transferred, as such?

Yes I had in mind exactly something like that. So creator would be able to create a collection of existing NFTs, or add new NFT to existing collection. Each NFT inside would be sold/bought/transferred individually. 

> If so, then it would seem to me that this use case only really works when there is more than 1 edition of each NFT, otherwise, the olny way to actually have a full collection is to get the exact single version of each video in the entire collection. It seems this kind of process can easily get stuck, with lots of bargaining costs and communication complexity.

I wouldn't really agree with that, that we would need to have more that 1 edition of each NFT. 
I don't think that especially with NFT collections people tend to "Catch them all", I think with most collections out there it also pretty impossible. If we will once again look for eg. at cryptopunks collection, they released 10.000 unique, 1 edition avatars. It's impossible for one person to get all of them but people still may want to be part of this and own at least one. 

So I think for having a collections of sort, we don't need to change 1 edition NFTs we will have now. 
If those collections would be created manually (so users selects NFTs and creates custom collection) or  collections would pull the name from playlists, or some different approach I still think it's something we may consider having in future.

As for the Curator groups managing channels it would be great if we could maybe talk about it on Monday, at the end of our weekly call. I think other designers would also like to get to know that concept, I think it was pretty new infor for everyone :D

**Comment by bedeho** [comment]
> So I think for having a collections of sort, we don't need to change 1 edition NFTs we will have now.

Ok, that is very interesting. I would love to understand the value of this concept to users, do you have some mental model for this?

> As for the Curator groups managing channels it would be great if we could maybe talk about it on Monday, at the end of our weekly call. I think other designers would also like to get to know that concept, I think it was pretty new infor for everyone :D

Will add to agenda!

**Comment by Lenorette** [comment]
@bedeho 
I see many values in this approach, for example, creators could postpone the release of NFTs and release all of them once they are done with the series. Quite often creators start small and slowly grow the audience. The possibility of selling series of NFTs for successful series of movies could drive up engagement with the platform. If they decide to release NFT as soon as a video is up they could hype up upcoming auctions through different channels ie. social media. In case of releasing NFT right after upload, it could also help creators to decide which series they should explore more.
Collectors would also benefit from that, the whole series of NFTs from your favorite artists would be more valuable for them than a few NFTs without any connection to each other. It could also reinforce the second-hand market - seeing that you are missing few movies could drive you to seek them on auctions.

**Comment by bedeho** [comment]
Two questions

- once an NFT is added to a collection, can it be remove also?
- once a collection has been created, can it later be removed as a collection, while the underlying individual NFTs live on?

> I see many values in this approach, for example, creators could postpone the release of NFTs and release all of them once they are done with the series.

This sounds useful, but I am not sure why collections are needed to achieve this?

> If they decide to release NFT as soon as a video is up they could hype up upcoming auctions through different channels ie. social media.

Same question here.

> Collectors would also benefit from that, the whole series of NFTs from your favorite artists would be more valuable for them than a few NFTs without any connection to each other.

Ok yes, this makes sense to me. The NFT items come together to make some unified whole of creative work which has a theme or meaning which has a life of its own and can spread virally, more effectively than any given NFT.

**Comment by Lenorette** [comment]
@bedeho 
> - once an NFT is added to a collection, can it be remove also?
> - once a collection has been created, can it later be removed as a collection, while the underlying individual NFTs live on?

I think there are two options for that, and they both depend on how NFTs will be implemented. 

1. There will be only option to add to collection, without removing from  and deleting collections. So if something is done it's done and it's not reversible, same as when we create and sell NFT there is no option to delete it. 
2. We could treat collections similarly to Categories now (music, sport, ect..), but it would be optional field. It would mainly give collectors a better way to filter existing NFTs and look for a specific group/collection. In this case collections could be changed, deleted, they even could be connected to playlist if we would have them in future. So if creator has a playlist created, NFT could pull the name of the playlist as a "collection", so it would belong to specific series of videos. 


As for the rest. I think what you've written is the exact reason for all questions.
> The NFT items come together to make some unified whole of creative work which has a theme or meaning which has a life of its own and can spread virally, more effectively than any given NFT.

I mentioned in the beginning of our conversation, I don't think collections are necessary thing for a MVP but in future it may be valuable for collectors. Of course hyping release of NFT on social media, or releasing group of NFTs at the same time can be done without having collections. But having it in some sort of collection can be more meaningful for collectors.

**Comment by bedeho** [comment]
> I think there are two options for that, and they both depend on how NFTs will be implemented.

The first option seems more powerful to me, so without any more information I would opt for that.

> I don't think collections are necessary thing for a MVP but in future it may be valuable for collectors.

Agreed, perhaps we should start collecting these ideas of future NFT ideas somewhere, its going to get exceedingly hard to remember otherwise as we tackle more and more topics.
